src/main/java/com/hcl/appscan/jenkins/plugin/scanners/StaticAnalyzer.java

```diff
@@ -32,29 +32,48 @@ public class StaticAnalyzer extends Scanner {
         
         private boolean m_openSourceOnly;
         private boolean m_sourceCodeOnly;
+        private String m_scanSpeed;
         
         @Deprecated
         public StaticAnalyzer(String target){
             this(target,false);
         }
         
-        public StaticAnalyzer(String target, boolean hasOptions, boolean openSourceOnly, boolean sourceCodeOnly){
```

Comment:
We should keep the old constructor using @Deprecated and pass null when/if it's called.

Reply:
If I recall correctly, we did not make this modification when initially adding the open source or source code only. Are we considering this change due to the repository being public and the possibility that it could affect users who have made local changes?

Reply:
If we don't need it to retain backwards compatibility, then that should be ok. Have we tested running existing jobs after upgrading to this version of the plugin?

Reply:
Yes Matt, did the upgrade testing, getting the expected results after these changes.